tests/textcase/telescope/telescope_spec.lua

```diff
@@ -10,6 +10,9 @@ describe("Telescope Integration", function()
   before_each(function()
     textcase.setup({})
     require("telescope").load_extension("textcase")
+
+    local buf = vim.api.nvim_create_buf(false, true)
+    vim.api.nvim_command("buffer " .. buf)
```

Comment:
This is needed to separate the test runs so that marks don't leak into other tests.